src/components/shared/nav/nav.module.scss

```diff
@@ -0,0 +1,107 @@
+@use '@variables' as *;
+
+.container {
```

Comment:
There are a lot of container components here. Im sure they all have a purpose, but do you need them all, or is it possible to combine some of them? For example, can the properties of maxWidth be added to container? Can the properties of navUl be added to navigation ?